objcgen/objcgenerator.cs

```diff
@@ -367,6 +367,35 @@ protected override void Generate (Type t)
 				builder.WriteImplementation ();
 			}
 
+			if (equals.TryGetValue (t, out m)) {
+				var builder = new EqualsHelper (headers, implementation) {
+					ObjCSignature = $"isEqual:(id)other",
```

Comment:
no need for `$` here - even if, _I suspect_, the C# compiler won't call `Format` unless needed